examples/dswx_s1_sample_runconfig-v3.0.0-er.1.0.yaml

```diff
@@ -0,0 +1,200 @@
+# Sample RunConfig for use with the DSWx-S1 PGE
+# This RunConfig should require minimal changes in order to be used with the
+# OPERA PCM.
+
+RunConfig:
+    # Name for the RunConfig, may be any string
+    Name: OPERA-DSWX-S1-PGE-SAMPLE-CONFIG
+
+    Groups:
+        # PGE-specific RunConfig section
+        # This section is only used by the PGE, however, paths to inputs/outputs
+        # should align with the similar sections of the SAS RunConfig
+        PGE:
+            PGENameGroup:
+                # Name of the PGE for use with this RunConfig, should always be
+                # DSWX_S1_PGE when using with the DSWx-S1 PGE
+                PGEName: DSWX_S1_PGE
+
+            InputFilesGroup:
+                # List of input files/directories
+                # Must be either a top-level directory containing one or more HLS
+                # .tif files, or a list of all HLS .tif files for use with the job
+                # Paths must correspond to the file system within the Docker container
+                InputFilePaths:
+                    - /home/conda/input_dir
+
+            DynamicAncillaryFilesGroup:
+                # Map of ancillary file types to paths to the file
+                # Paths must correspond to the file system within the Docker container
+                AncillaryFileMap:
+                    # Path to the Digital Elevation Model
+                    # Must be either a single .tif, or a .vrt that points to one
+                    # or more .tif files
+                    dem_file: /home/conda/input_dir/dem.tif
+
+                    # Path to the Discrete Classification Map
+                    # Must point to a single .tif file
+                    hand_file: /home/conda/input_dir/hand.tif
+
+                    reference_water_file: /home/conda/input_dir/reference_water.tif
+
+                    worldcover: /home/conda/input_dir/worldcover.tif
```

Comment:
The key should match what the SAS uses, for consistency. So this should be `worldcover_file`

Reply:
Nice catch Scott.

Reply:
Looks like this got updated to `world_cover`, and not `worldcover_file`...

Reply:
I changed it, but it looks like it's worldcover_file in dsw_s1.yaml from the download.

Reply:
I also changed it in the schema file

Reply:
Ah I see, I had not realized that the SAS schema was also calling this filed "world_file", thanks for addressing